src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -837,7 +837,7 @@ public virtual bool HasUnsupportedMetadata
         /// PROTOTYPE(NullableReferenceTypes): presumably whether the feature is turned on or off will affect NonNullTypes default value on source module.
         /// Not valid to call on non-definitions.
         /// </summary>
-        internal virtual bool NonNullTypes
+        public virtual bool NonNullTypes
```

Comment:
>public [](start = 8, length = 6)

Why `public`?

Reply:
It's `public` to implicitly implement the `INonNullTypesContext` interface.